### DISCUSSION TITLE
Disable X axis tooltip but keep y axis tooltip title

### DISCUSSION THREAD

**Discussion by ivocarbajo** [apexcharts/apexcharts.js#3130] [Q&A] [answered]
Hello, I'm looking for a way to disable the tooltip marked in red whilst keeping the one marked in blue, or a way to add a tooltip item with a value calculated from the other items that doesn't involve rebuilding the entire tooltip.
For now I am using the tooltip title (x axis tooltip) to display my "custom item" but it's suboptimal... It works though, so I wouldn't mind if at least I could get rid of the x axis tooltip.
Thank you! :)
![image](https://user-images.githubusercontent.com/58275681/166893737-a61fd8a3-5e94-4cb7-9d12-e90553568f7f.png)

**Comment by csanchezarisa** [comment, marked as answer]
Hello!

You can modify this tooltip by using the xaxis.tooltip property.
[apexcharts doc](https://apexcharts.com/docs/options/xaxis/)

If you want to disable it:
```ts
your_chart.xaxis = {
    enabled: false
}
```

**Reply by ivocarbajo** [reply]
Amazing, that works, thank you!
